enterprise/migrations/0140_update_enrollment_sources.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+
+ENTERPRISE_ENROLLMENT_SOURCES = {
+    'Admin Enterprise Enrollment': 'customer_admin'
```

Comment:
Name wise, Enterprise Admin Enrollment makes more sense to me, but open to thoughts (that's just me!)

Reply:
Or just `Customer Admin Enrollment`, since that is the name used all over anyway